### PR TITLE
[v7r1] Fix group by syntax in ReqSummaryWeb

### PR DESCRIPTION
**Pull request by arrabito**
*RequestManagement
FIX: Fix the GROUP BY syntax in the RequestSummaryWeb to be compliant with MariaDB 10.5. The fix is backward compatible with MySQL 5.7

### CODE REVIEW

**Comment by arrabito** [comment]
Any suggestion how I can fix the code to pass the test or how I can rerun the test on my side?

**Comment by atsareg** [comment]
After rerunning the tests it looks OK now

**Review by atsareg — DISMISSED** [review]

File: RequestManagementSystem/DB/RequestDB.py [hunk 1]
```diff
@@ -575,7 +575,10 @@ def getRequestSummaryWeb(self, selectDict, sortList, startItem, maxItems):
 
           if key == 'Type':
             summaryQuery = summaryQuery.join(Request.__operations__)\
-                                       .group_by(Request.RequestID, Operation.Type)
+                                       .group_by(Request.RequestID, Request.RequestName,
+                                                 Request.JobID, Request.OwnerDN, Request.OwnerGroup,
+                                                 Request._Status, Request.Error,
+                                                 Request._CreationTime, Request._LastUpdate)
```
Comment (fstagni) [inline RequestManagementSystem/DB/RequestDB.py:581]: The grouping by `Operarion.Type` is gone? why?

Comment (chaen) [reply RequestManagementSystem/DB/RequestDB.py:581]: it should stay clearly

Comment (arrabito) [reply RequestManagementSystem/DB/RequestDB.py]: Yes. Just fixed.

**Comment by chaen** [comment]
Seems ok but there is a problem with the doc, you probably need to rebase:

```

WARNING: autodoc: failed to import module u'RucioSynchronizerAgent' from module u'DIRAC.ConfigurationSystem.Agent'; the following exception was raised:
No module named enum
WARNING: autodoc: failed to import module u'RucioRSSAgent' from module u'DIRAC.ResourceStatusSystem.Agent'; the following exception was raised:
No module named enum
WARNING: autodoc: failed to import module u'RucioFileCatalogClient' from module u'DIRAC.Resources.Catalog'; the following exception was raised:
No module named enum
```

**Review by fstagni — APPROVED** [review]

**Pull request merged**